src/orquestra/sdk/_ray/_dag.py

```diff
@@ -562,6 +563,7 @@ class RayParams:
     log_to_driver: bool = True
     storage: t.Optional[t.Union[str, _client.Storage]] = None
     _temp_dir: t.Optional[str] = None
+    configure_logging: bool = True
```

Comment:
**thought**: you might need to also extend some test fixtures. I can't remember now (sorry, it's late!) but if the CI failure logs will contain hints.